packages/api/src/api/helix/moderation/HelixModerationApi.ts

```diff
@@ -437,4 +440,59 @@ export class HelixModerationApi extends BaseApi {
 			}
 		});
 	}
+
+	/**
+	 * Gets the broadcaster's Shield Mode activation status.
+	 *
+	 * @param broadcaster The broadcaster whose Shield Mode activation status you want to get.
+	 * @param moderator A user that has permission to read Shield Mode status in the broadcaster's channel.
+	 * This user must match the user associated with the user OAuth token.
+	 *
+	 * @beta
+	 */
+	async getShieldModeStatus(
+		broadcaster: UserIdResolvable,
+		moderator: UserIdResolvable
+	): Promise<HelixShieldModeStatus> {
+		const result = await this._client.callApi<HelixResponse<HelixShieldModeStatusData>>({
+			type: 'helix',
+			url: 'moderation/shield_mode',
+			method: 'GET',
+			scope: 'moderator:read:shield_mode',
+			query: {
+				...createModeratorActionQuery(broadcaster, moderator)
```

Comment:
No need to spread into an otherwise empty object - just pass it as query directly

Reply:
Fixed